formatTest/unit_tests/expected_output/ocaml_identifiers.re

```diff
@@ -10,22 +10,22 @@ module V = {
 
 /* Record fields */
 module R = {
-  type r = {mutable method: int};
+  type r = {mutable method_: int};
```

Comment:
Asked offline. I didn't think that you wanted to revert the record field mangling. Was that intentional?

Reply:
With records being used in place of objects, a difference in mangling would come up pretty soon.
To be on the safe side, I've kept refmt from bs 6.2.1 as a reference.
There are still a couple of difference to take into account, if one wants to have something that behaves like a revert. E.g. mangling of the name in type declarations.

Reply:
On the native side we pretty much want all the mangling we can get for greater compatibility. On the bs side, you can disable whatever you like under a flag. I just wasn't sure if you wanted something that behaved like a perfect revert. If you're fine with it, go ahead and merge.